src/main/kotlin/calculator/Convertor.kt

```diff
@@ -0,0 +1,22 @@
+package calculator
+
+class Convertor {
+    /**
+     * 입력된 수식 문자열을 화이트스페이스를 기준으로 split하여 리스트로 반환한다.
+     */
+    fun splitNumberSentenceToList(numberSentence: String?): List<String> {
+        if (numberSentence == null || numberSentence.isBlank() || numberSentence.isEmpty())
+            throw IllegalArgumentException("입력값이 null이거나 빈 공백이 아니여야합니다.")
+
+        val splitByWhitespaceList = numberSentence.split(" ")
+        if (splitByWhitespaceList.size < 3) throw IllegalArgumentException("숫자 및 기호를 포함하여 3개 이상을 입력하셔야합니다.")
+        if (splitByWhitespaceList.first().toIntOrNull() == null || splitByWhitespaceList.last().toIntOrNull() == null)
+            throw IllegalArgumentException("수식의 시작 및 끝은 숫자여야합니다.")
```

Comment:
두개의 코드가 서로다른 포맷을 하고 있는데요. 
동일한 코드 컨벤션을 가져가는게 읽기에 좋지 않을까요?  
```suggestion
        if (splitByWhitespaceList.size < 3) {
          throw IllegalArgumentException("숫자 및 기호를 포함하여 3개 이상을 입력하셔야합니다.")
        }
        if (
          splitByWhitespaceList.first().toIntOrNull() == null || 
          splitByWhitespaceList.last().toIntOrNull() == null
        ) {
            throw IllegalArgumentException("수식의 시작 및 끝은 숫자여야합니다.")
        }
```

추가로 여러개의 조건문이 if 절에 들어갈때는 [코틀린 코딩 컨벤션이](https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements) 존재합니다~